### PR TITLE
Add instructions for configuring VS Code

### PR DESCRIPTION
**Pull request by mijo-gracanin**
Add instructions for configuring VS Code to build Godot

### CODE REVIEW

File: development/cpp/configuring_an_ide.rst [hunk 1]
```diff
@@ -258,3 +259,42 @@ Test it:
...
+
+
+.. _doc_configuring_an_ide_vscode:
+
+Visual Studio Code
+------------------
+
+- Ensure that C/C++ extension is installed. You can find instructions in `docs <https://code.visualstudio.com/docs/languages/cpp>`_.
+
+- Now open cloned godot folder in VS Code (File > Open Foleder...)
```
Comment (aBARICHELLO) [inline development/cpp/configuring_an_ide.rst]: Typo: *Foleder* -> Folder

Comment (mijo-gracanin) [reply development/cpp/configuring_an_ide.rst]: Thank you! 
Updated

**Comment by toger5** [comment]
Nice ! I like it

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**